bundles/framework/divmanazer/instance.js

```diff
@@ -1066,6 +1040,10 @@ Oskari.clazz.define('Oskari.userinterface.bundle.ui.UserInterfaceBundleInstance'
             restore: function (flyout, flyoutPlugin, extensionInfo) {
                 var me = this;
                 //    viewState;
+                 /*
+                 * to top
+                 */
+                me.shuffleZIndices(flyout);
```

Comment:
flyout.bringToTop() instead? Though shuffleZIndices() resets the z-indexes and bringToTop() just increases the value so after some time flyouts start to have higher z-indexes than the overlay-component?

Reply:
The z-value does have a 80k margin so I think we are pretty safe even if we don't reset.